libmozdata/hgmozilla.py

```diff
@@ -241,3 +246,41 @@ def get(paths, channel='nightly', node='tip'):
             FileInfo(queries=queries).wait()
 
         return data
+
+
+class HGMozilla(object):
```

Comment:
Doesn't this class do something really similar to HGFileInfo? Perhaps we should move them together in the same file?
I wish we could have a single interface for them, so you don't have to write different code to use this one vs the other one.